bitnami/mongodb-sharded/templates/mongos/mongos-service-per-replica.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.mongos.useStatefulSet .Values.mongos.servicePerReplica.enabled }}
+{{- range $i := until (.Values.mongos.replicas | int) }}
+{{- $context := merge $ (dict "index" $i) }}
```

Comment:
I think this is also not needed since you can use `$` as `$context`.
Is there any reason to merge the index to the context?

Reply:
The index is used to templatize the annotations (eg. `external-dns.alpha.kubernetes.io/hostname: mongos-{{ .index }}.example.com`).
If we use `$` directly there's no way to know the current index in the template but it's needed to create a unique dns record for example.

Reply:
👍